atoma-sui/src/subscriber.rs

```diff
@@ -446,6 +446,8 @@ fn write_cursor_to_toml_file(cursor: Option<EventID>, path: &str) -> Result<()>
 /// * `StackSettlementTicketEvent` - Handles stack settlement tickets
 /// * `StackSettlementTicketClaimedEvent` - Handles claimed stack settlement tickets
 /// * `StackAttestationDisputeEvent` - Handles stack attestation disputes
+/// * `NewKeyRotationEvent` - Handles new key rotation requests
+/// * `NodeKeyRotationEvent` - Handles node key rotation events
```

Comment:
question - what is difference between these two?

Reply:
The event `NewKeyRotationEvent` relates to a new epoch key rotation event, applied to every node.

The event `NodeKeyRotationEvent` is emitted whenever a node submits a transaction with a new generated public key. So that the entire network is aware that the given node has committed to a new public key.

@dayeol does this make sense to you ?